diff --git a/.cargo/config b/.cargo/config
index 5a96b1f..4450076 100644
--- a/.cargo/config
+++ b/.cargo/config
@@ -10,9 +10,10 @@ rtv7em = "rthumbv7em"
 [target.'cfg(any(target_arch = "arm", target_arch = "riscv32"))']
 rustflags = [
     "-C", "relocation-model=epic",
+    "-C", "target-feature=+relax",
+    "-C", "link-arg=--relax",
     "-C", "link-arg=--lto-epic",
     "-C", "link-arg=--emit-relocs",
     "-C", "link-arg=-Tlayout.ld",
-    "-C", "linker-plugin-lto",
 ]
 runner = ["cargo", "run", "-p", "runner", "--release"]
